### DISCUSSION TITLE
Manage bonds with MMEX

### DISCUSSION THREAD

**Discussion by frankieorabona** [moneymanagerex/moneymanagerex#6295] [Q&A]
I can't figure out how to manage some Bonds ( e.g BTP Italy)
In which account category should I put them? And how do I manage any losses or profits due to market price?

For example, I had purchased another ROM-38 bond at 71.84 and now it has dropped to 68.90. How do I handle these losses? Suggestions?

By the way, how can I manage coupons (semi-annual or annual)? For most of the Bonds knowing the rate I could derive them already with a PLANNED TRANSACTION, but I would like it to always be tied to the account I create and not to other accounts (making for example transfers -for purchase or for sale- as I currently do since then I would have no entry or exit entries for reporting)

**Comment by frankieorabona** [comment]
I am beginning to think that I am the only one who invests in bonds  :)

Anyone who has any suggestions for me? Maybe post me an example of it?

**Reply by vomikan** [reply]
Investments have been implemented very superficially.
There is no direct support for bonds. 
If desired, you can, of course, upload quotes in hand mode and make transfers like coupons from asset accounts... But so few people do that.

**Reply by frankieorabona** [reply]
First of all, thank you for responding to me! Reading from the description of the accounts that they could be used for bonds I thought they were implemented for that. Even if I wanted to follow your suggestion, I really have no idea how to do it... for me MMEX still has a lot of functionality unknown to me. It is a great tool, but it suffers from a lack of guides.... it should be a little more user friendly

**Reply by vomikan** [reply]
I found an old video in which the developer explains the concept.
https://youtu.be/UJr2M743p10

**Reply by frankieorabona** [reply]
Well...I thank you for the video! Unfortunately, this video is for stocks. For bonds I don't know how much sense it would make. I don't have a number of bonds to buy, because it doesn't have a price of a single bond...I decide the sum of the bonds I want to buy ( usually in multiples of 100 or 1000) versus the stocks.

**Reply by vomikan** [reply]
I don't see a fundamental difference between stocks and bonds. Some stocks are also sold in lots.
 Stocks have dividends, bonds have coupons. 
The share can be sold, the bond has a maturity date.
But in any case, the functionality of MMEX in this sense is very limited.
It is only possible to acquire assets known to Yahoo and observe the change in the value of the portfolio.

**Reply by frankieorabona** [reply]
Thank you again for your reply.

The problem is that the functionality is really limited.
However, I might be the one who has not understood some of the functionality of MMEX (in which case I apologise for this and would ask you to show me how to do this, even with a test DB).

I'll give you 1 example:
I buy 950 euro bond from the default account. How do I register this transaction? Transfer? But in that case I will not see my expenditure in the various reports. I understand that it is a transfer of funds, but I currently have a cash outlay that I will not see in the various reports.

After one year, a coupon accrues. The amount is credited to the default account.  How do I record this entry? Individually and independently of the investment account? I wouldn't know how to link it. Well... I would like to see how much the bond is earning me on MMEX, if I have to keep a separate note, I might as well use EXCEL sheets to monitor it.

After two years the bond is redeemed at 1000, how do I record the return of the initial 950 (transfer? The same as buying, but in reverse) but above all the 50 euro gain?

MMEX is perfect in many things, but on other things it sins in an absurd way (see for example also the management of loans)

(Sorry for my bad english)
